scripts/ci-build-azure-ccm.sh

```diff
@@ -61,13 +61,8 @@ main() {
     if [[ "$(can_reuse_artifacts)" =~ "false" ]]; then
         echo "Build Linux Azure amd64 cloud controller manager"
         make -C "${AZURE_CLOUD_PROVIDER_ROOT}" build-ccm-image-amd64 push-ccm-image-amd64
-        if [[ -n "${TEST_WINDOWS:-}" ]]; then
-            echo "Building Linux amd64 and Windows ${WINDOWS_IMAGE_VERSION} amd64 cloud node managers"
+        echo "Building Linux amd64 and Windows ${WINDOWS_IMAGE_VERSION} amd64 cloud node managers"
```

Comment:
should we build the image for both Windows versions right away? or only if we end up needing it?  Could go either way.

Not as worried about appending the additional image to the manifest list as I was when we were converting a single arch image to manifest list.

Reply:
I think what I'm learning in this PR is that "if we end up needing it" is not something we can deterministically predict, and so building "everything that we may need" is a better approach

Reply:
We normally don't set `WINDOWS_SERVER_VERSION` when we are testing WS2019. I think that would result in `WINDOWS_IMAGE_VERSION` not being set (based on lines 49-57 above)

Reply:
can we set an appropriate default for `WINDOWS_IMAGE_VERSION` if both it and `WINDOWS_SERVER_VERSION` are not set?

Reply:
I assume that would be something like `export WINDOWS_IMAGE_VERSION="1809"` given what we're doing above

Reply:
Yea we can do that.
Can you please also refernce https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/8cc82a6cdbe13f38eba040249c0b51d9824e38c7/azure/defaults.go#L61-L65
So if/when we do switch the default from WS2019 to WS2022 we know all the places we are defaulting to 2019?

Reply:
actually I think L38 is already doing this for us

I can add a reference

Reply:
Oh yea, I didn't scroll up enough in the file.
Let's add the ref and then this should be good to merge.

Reply:
Given the state of E2E and the importance of this if it passes current tests I think we should merge and then I'll submit a follow-up PR w/ code comments and such. Stay tuned...